lib/ipmi_sensor.c

```diff
@@ -260,6 +260,183 @@ print_thresh_setting(struct sdr_record_full_sensor *full,
 	}
 }
 
+static void
+dump_sensor_fc_thredshold_csv(int thresh_available,
+			 const char *thresh_status,
+			 struct ipmi_rs *rsp,
+			 struct sensor_reading *sr)
+{
+	printf("%s", sr->s_id);
+	if (sr->s_reading_valid) {
+		if (sr->s_has_analog_value)
+			printf(",%.3f,%s,%s",
+				   sr->s_a_val, sr->s_a_units, thresh_status);
```

Comment:
This is also strangely indented/aligned. Need 1 TAB less and spaces for alignment. Same applies to many lines below.